lib/Fhp/FinTs.php

```diff
@@ -149,8 +149,7 @@ public function getAccounts(\Closure $tanCallback = null)
 			$dialog,
 			array(
 				new HKIDN(3, $this->bankCode, $this->username, $dialog->getSystemId()),
-				new HKVVB(4, 0, 0, HKVVB::LANG_DE, $this->productName, $this->productVersion),
-				$this->createHKTAN(5)
```

Comment:
Hatten wir nicht ein paar `HIPINS` angeschaut, und `HKIDN` bzw. `HKVVB` waren dort nicht "korrekt" gelistet? Mag mich täuschen.

Außerdem sagt [Abschnitt B.4.3.1](https://www.hbci-zka.de/dokumente/spezifikation_deutsch/fintsv3/FinTS_3.0_Security_Sicherheitsverfahren_PINTAN_2018-02-23_final_version.pdf) in der grauen Box, dass man immer `HKTAN` bei der Dialoginitialisierung mitschicken soll, einfach um zu signalisieren, ob PSD2 unterstützt wird. Im Zweifel eben ein hauptsächlich leeres `HKTAN`.

Reply:
getAccounts scheint ein Sonderfall zu sein ohne eigenen Geschäftsvorfall? D.h. es wird eigentlich eine Dialog Initialisierung durchgeführt?